fusedrug/data/tokenizer/modulartokenizer/pretrained_tokenizers/bmfm_extended_modular_tokenizer/cell_attributes_tokenizer.json

```diff
@@ -2963,6 +3323,46 @@
       "<CORRUPTED_AREA_END>": 291,
       "<MOLECULAR_ENTITY_MUTATED_PROTEIN_CHAIN>": 292,
       "<MOLECULAR_ENTITY_PROTEIN_CHAIN>": 293,
+      "<10>": 294,
+      "<11>": 295,
+      "<12>": 296,
+      "<13>": 297,
+      "<14>": 298,
+      "<15>": 299,
+      "<16>": 300,
+      "<17>": 301,
+      "<18>": 302,
+      "<19>": 303,
+      "<20>": 304,
+      "<21>": 305,
+      "<22>": 306,
+      "<23>": 307,
+      "<24>": 308,
+      "<25>": 309,
+      "<26>": 310,
+      "<27>": 311,
+      "<28>": 312,
+      "<29>": 313,
+      "<30>": 314,
+      "<31>": 315,
+      "<32>": 316,
+      "<33>": 317,
+      "<34>": 318,
+      "<35>": 319,
+      "<36>": 320,
+      "<37>": 321,
+      "<38>": 322,
+      "<39>": 323,
+      "<40>": 324,
+      "<41>": 325,
+      "<42>": 326,
+      "<43>": 327,
+      "<44>": 328,
+      "<45>": 329,
+      "<46>": 330,
+      "<47>": 331,
+      "<48>": 332,
+      "<49>": 333,
```

Comment:
Why do we need numbers? Can't we just use digits? Like <1>, <0> instead of <10>?

Reply:
In any case, I'm not sure special tokens is the place for integers. We only have 500 special tokens, and this is eating them up...